spec/unit/lib/cadence/workflow/serializer/query_answer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'cadence/workflow/serializer/query_failure'
+require 'cadence/workflow/query_result'
+require 'cadence/workflow/serializer/query_answer'
+
+describe Cadence::Workflow::Serializer::QueryAnswer do
+  class TestDeserializer
+  end
+
+  describe 'to_proto' do
+    let(:query_result) { Cadence::Workflow::QueryResult.answer(42) }
+    it 'produces a protobuf' do
```

Comment:
Change this to `it produces a thrift object`. (Cadence uses thrift instead of protobufs)